azurerm/internal/services/web/app_service.go

```diff
@@ -812,7 +812,13 @@ func schemaAppServiceIpRestriction() *schema.Schema {
 				"ip_address": {
 					Type:         schema.TypeString,
 					Optional:     true,
-					ValidateFunc: validation.IsCIDR,
+					ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
we should revert this?

Reply:
Of course, I was intending to do this but forgot, reverted now.